nx/lib/nx.ex

```diff
@@ -16823,11 +16823,11 @@ defmodule Nx do
 
       t = Nx.u8([10, 20, 30])
       Nx.to_pointer(t, mode: :local)
-      %Nx.Pointer{kind: :local, address: 1234, data_size: 3, handle: nil}
+      #=> {:ok, %Nx.Pointer{kind: :local, address: 1234, data_size: 3, handle: nil}}
```

Comment:
Btw. in retrospect we should probably return a pointer and raise on failure, because an error is likely a developer error, and the error EXLA returns is just a string, so it's pretty opaque. That would be breaking, so not sure. Maybe as part of v1?

cc @polvalente

Reply:
Ok, I will send another PR soon!